Tests/Form/Widget/FormSonataFilterChoiceWidgetTest.php

```diff
@@ -64,7 +63,7 @@ public function testDefaultValueRendering()
 
     protected function getParentClass()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
+        if (class_exists('Symfony\Component\Form\Extension\Core\Type\RangeType')) {
```

Comment:
Precisely. I'm not saying it is better, but if it is not worse, then we would probably be better off using always the same test I guess…